src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int ERROR = -1;
+    private static final int MAX_STACK = 12;
+    private static final int CAPACITY = 11;
```

Comment:
This constant is related to MAX_STACK. Changing only one leads to an error (inconsistency).